e2e-tests/functions

```diff
@@ -740,8 +783,17 @@ destroy() {
 	kubectl_bin delete -f "https://github.com/jetstack/cert-manager/releases/download/v${CERT_MANAGER_VER}/cert-manager.yaml" 2>/dev/null || :
 	if [ -n "$OPENSHIFT" ]; then
 		oc delete --grace-period=0 --force=true project "$namespace" &
+		if [ -n "$OPERATOR_NS" ]; then
+			oc delete --grace-period=0 --force=true project "$OPERATOR_NS" &
+			if [ "${OPENSHIFT}" == "3.11" ]; then
```

Comment:
we do not support OPENSHIFT 3.11

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/973/commits/ab103b54ca7a80b80804e95cb2d9149bbcb45417